### PR TITLE
Feat: Logout dropdown

### PR DESCRIPTION
**Pull request by rukundo-kevin**
# PR Description
 Logout dropdown
# Description of tasks that were expected to be completed

1.  users are able view a dropdown when they click their profile pic or avatar.

# functionality 

- Sign in with an email e.g(admin@random.com) through this [link]( https://devpulse-review-app-61.herokuapp.com/), use any characters of your choice as password 
- Click the profile avar on top-right corner.

# Screenshots (If appropriate)
Screen Shot 2022-08-19 at 11 07 10
Screen Shot 2022-08-19 at 11 07 32
Screen Shot 2022-08-19 at 11 07 53

# Please check this Checklist before you submit your PR:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My code generate no warnings
- [x] My test coverage meet the set test coverage threshold
- [x] There are no vulnerabilities
- [x] There are no conflicts with the base branch

### CODE REVIEW

**Comment by feyton** [comment]
I deployed a review app for you to try out this pull request 👇🏽

Heroku App: https://devpulse-review-app-61.herokuapp.com/


<hidden purpose="for-rewritable-pr-comment-action-use" value="Deploy"></hidden>

**Review by chanceDira — APPROVED** [review]
Nice job.

**Comment by IIshimwe** [comment]
Doesn't it look better if you remove color changes on the text and replace it with color changes on the background for better visibility of what hovered on?

File: src/components/ProfileDropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            data-testid="notificationsContainer"
+          >
+            <div className="w-full p-3 pl-4 border-border-dark dark:border-white border-b-[0.5px]">
+              <div className="flex flex-row justify-between align-center gap-x-[20px] ">
+                <div className="flex flex-col w-full gap-[5px] cursor-pointer">
+                  <p className="font-semibold text-gray-600 dark:text-white p-1 hover:bg-gray-600 hover:text-gray-200 dark:hover:bg-gray-300 dark:hover:text-gray-900">
+                    <Link to="/dashboard/profile">{t('Profile')}</Link>
+                  </p>
+                  <p className="font-semibold text-gray-600 dark:text-white p-1 hover:bg-gray-600 hover:text-gray-200 dark:hover:bg-gray-300 dark:hover:text-gray-900">
+                    <Link to="#link">{t('Preference')}</Link>
```
Comment (feyton) [inline src/components/ProfileDropdown.tsx:44]: You can change the link to point to the `settings` page

File: src/components/ProfileDropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            </p>
+          </div>
+          <div
+            className="flex flex-col w-full overflow-auto"
+            data-testid="notificationsContainer"
+          >
+            <div className="w-full p-3 pl-4 border-border-dark dark:border-white border-b-[0.5px]">
+              <div className="flex flex-row justify-between align-center gap-x-[20px] ">
+                <div className="flex flex-col w-full gap-[5px] cursor-pointer">
+                  <p className="font-semibold text-gray-600 dark:text-white p-1 hover:bg-gray-600 hover:text-gray-200 dark:hover:bg-gray-300 dark:hover:text-gray-900">
```
Comment (feyton) [inline src/components/ProfileDropdown.tsx:40]: Give the `menu` the full width to fix this issue of hover effects different for the `signout` to the others

image

**Review by feyton — CHANGES_REQUESTED** [review]
### Request changes
@rukundo-kevin 
Please look into the comments I left for some minor fixes.
Kudos for the great feature that was really needed

**Review by feyton — APPROVED** [review]

**Review by IIshimwe — APPROVED** [review]
This is nice @rukundo-kevin

**Review by ceelogre — APPROVED** [review]

**Pull request merged**